Fixes error where getLoadOnlyIRIPrefix is null

markquintontulloch opened the pull request:
Was getting the following error:
java.lang.NullPointerException: Cannot invoke "String.length()"
because "prefix" is null

oblodgett commented on src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java:
```diff
@@ -100,7 +100,7 @@ public T traverse(OWLClass parent, int depth, ArrayList<String> requiredNamespac
             if(
                 (termParent.getNamespace() != null && requiredNamespaces.contains(termParent.getNamespace())) ||
-                (parent.getIRI().getShortForm().startsWith(config.getLoadOnlyIRIPrefix()))
+                (config.getLoadOnlyIRIPrefix() != null && parent.getIRI().getShortForm().startsWith(config.getLoadOnlyIRIPrefix()))
```
Oh... i guess that produces a null pointer. Sorry about that.

markquintontulloch commented:
@oblodgett - I think the above commit addresses most of the issues.  The refactor makes things easier to understand, to my mind at least, and I think I'm now doing the correct thing with regards to checking entities that are referred to are in the database and retrieving/setting the full objects.
